python/cugraph/centrality/betweenness_centrality_wrapper.pyx

```diff
@@ -26,82 +27,227 @@ import cudf
 import numpy as np
 import numpy.ctypeslib as ctypeslib
 
+import dask_cudf
+import dask_cuda
+import cugraph.raft
 
-def get_output_df(input_graph, result_dtype):
-    number_of_vertices = input_graph.number_of_vertices()
+
+import cugraph.comms.comms as Comms
+from cugraph.dask.common.mg_utils import get_client
+from cugraph.raft.dask.common.comms import worker_state
```

Comment:
these can be accessed as https://github.com/rapidsai/cugraph/blob/38b032efa630ae01e29b8c73b81c34bfa3f5a905/python/cugraph/dask/link_analysis/pagerank.py#L24 instead of raft import

Reply:
Updated in 09cbb8c